src/core/css/inc/mixins.scss

```diff
@@ -162,6 +162,27 @@ $default-arrow-margin: 3px;
   }
 }
 
+@mixin arrow-big-blue($margin: $default-arrow-margin) {
```

Comment:
This doesn't seem like something that would be used very often. Can it live in `SearchForm/styles.scss` for now?